wiki/Tournaments/GTS/IGTS_2022/en.md

```diff
@@ -0,0 +1,611 @@
+2. The loser's bracket will play on the same pool as the winner's bracket of the same weekend.
+3. The Qualifier pool will be different from all other rounds, as it will have a format of 2 NoMod maps, 1 Hidden map, 1 HardRock map, 1 DoubleTime map, and 1 FreeMod map.
+4. Each mappool consists of 6 sections: **NoMod, Hidden, HardRock, DoubleTime, FreeMod, and Tiebreaker**.
+5. Total maps per stage will be listed as follows:
+   - The Group stage mappool will consist of 14 maps.
+   - The Round of 16 and Quarterfinals mappools will consist of 15 maps.
+   - The Semifinals, Finals and Grand Finals mappools will consist of 16 maps.
+6. All stages will have 2 Hidden, HardRock, and DoubleTime maps, and 3 FreeMod maps.
+7. The FreeMod pool is played with FreeMod activated. Every player may pick a combination of Hidden and HardRock, but at least one player from each team must have at least one mod activated.
+8. Players cannot pick the same mod combination as their team member.
+9. The tiebreaker is played under FreeMod conditions, but it is not necessary to have a mod activated.
+10. The mappool for each stage will be showcased a week beforehand.
+
+### Scheduling instructions
+
+1. Every stage will be held **during the weekend**. It is, however, allowed to schedule it on another day, if no good time is found during the weekend and a referee is available.
+2. Matches are allowed to overlap if referees are available.
+3. A best effort will be made to fit schedules with applicable timezones. The schedule for each stage will be released the weekend prior to it.
+4. Reschedules will only be considered if both teams agree to a time. This needs to be done and notified to the tournament staff before **Thursday at 23:59 UTC** in the particular week when the match takes place.
+5. Reschedules may only be requested **by the team captain**.
+
+### Match regulations
+
+1. A private lobby will be created 10 minutes before a referee starts the match. Both teams' captains will be invited into the room before match time, and the referee or the captains will invite the other players into the room.
+2. If no staff or referee is available, the match will be postponed.
+3. If less than the minimum required players attend, the maximum time the match can be postponed is **10 minutes**.
+4. There is no penalty for not joining the room instantly at match time as long as it is within the postponed time.
+5. If neither of both teams have the minimum required players after 10 minutes, it will count as a loss for both teams in Group stage. In the double-elimination bracket, the highest seeded team will advance.
+6. The match will be played with Team VS and Score v2, and the name of the room must be: `IGTS 2022: ({Team Red}) vs ({Team Blue})`.
+7. Each team is allowed one warm-up but it must be **below 3 minutes**, and must be sent to the referee and streamer of the match one hour prior to the match's start time. Otherwise, the team will have no warmups.
```

Comment:
```suggestion
7. Each team is allowed one warmup but it must be **below 3 minutes**, and must be sent to the referee and streamer of the match one hour prior to the match's start time. Otherwise, the team will have no warmups.
```